test/test_pipeline_baseline.py

```diff
@@ -51,8 +51,8 @@ def check_calls_match(expected_calls, actual_calls):
 @mock.patch('subprocess.call')
 @mock.patch('subprocess.check_call')
 @mock.patch('os.chdir')
-def _test_baseline(task_name, test_name, language, output_dir, mock_chdir,
-                   mock_check_call, mock_call, mock_gradle_task):
+def _test_baseline(task_name, test_name, language, output_dir, extra_kwargs,
```

Comment:
That makes sense. I've been trying to figure out how to combine this change with the mock patches on this function, though. If extra_kwargs is an optional argument, it has to be the last parameter of the function definition, so we might have a header like `def _test_baseline(task_name, test_name, language, output_dir, mock_chdir, mock_check_call, mock_call, mock_gradle_task, extra_kwargs={})`. But the patches seem to patch starting from the last argument by default, so they try to patch extra_kwargs. How should I add the optional extra_kwargs argument so that the mocks still work?